spec/MongoTransform.spec.js

```diff
@@ -320,6 +320,9 @@ describe('parseObjectToMongoObjectForCreate', () => {
     done();
   });
 
+});
+
+describe_only_db('mongo')('parseObjectToMongoObjectForCreate', () => {
```

Comment:
> Do you think I should rollback this change??

You can roll it back, but that doesn't really change anything.

The edge to edge test would look like:

```
it('should match all strings that starts with string', (done) => {
  let object = new Parse.Object('Object');
  object.set('strings', ['the', 'brown', 'fox', 'jumps']);
  let object2 = new Parse.Object('Object');
  object2.set('strings', ['over', 'the', 'lazy', 'dog']);
  Parse.Object.saveAll([object, object2]).then(() => {
   return request.get({
      url: Parse.serverURL + 'classes/Object',
      json: true,
      query: ... // build the JSON query
   });
  }).then((results) => {
     expect(....) // TODO: validate the results
  });
}); 

```